internal/conf/helpers.go

```diff
@@ -10,3 +12,18 @@ func HasExternalAuthProvider(c conftypes.SiteConfigQuerier) bool {
 	}
 	return false
 }
+
+func GetDeduplicatedForksIndex() map[string]struct{} {
+	index := map[string]struct{}{}
+
+	repoConf := Get().Repositories
+	if repoConf == nil {
+		return index
+	}
+
+	for _, name := range repoConf.DeduplicateForks {
+		index[name] = struct{}{}
+	}
```

Comment:
If you use the Set as suggested above, this should become:
```suggestion
	index.Add(repoConf.DeduplicateForks...)
```